great_expectations/data_context/util.py

```diff
@@ -198,6 +200,16 @@ def mask_db_url(cls, url: str, use_urlparse: bool = False, **kwargs) -> str:
             )
         return cls._mask_db_url_no_sa(url=url)
 
+    @staticmethod
+    def _is_config_str(value: str) -> bool:
+        from great_expectations.datasource.fluent.config_str import ConfigStr
+
+        try:
+            ConfigStr.validate_template_str_format(value)
+            return True
+        except Exception:
```

Comment:
Can you catch ValueError here since that's what it looks like `validate_template_str_formate` raises?

Reply:
Done!